2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Configure stats pid to 592
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Mateo-drr\.config\wandb\settings
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Loading settings from S:\Cryp-Tor\wandb\settings
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'cryptolsmt.py', 'program_abspath': 's:\\cryp-tor\\cryptolsmt.py', 'program': 's:\\cryp-tor\\cryptolsmt.py'}
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_init.py:_log_setup():524] Logging user logs to S:\Cryp-Tor\wandb\run-20240329_220405-s82ool3r\logs\debug.log
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_init.py:_log_setup():525] Logging internal logs to S:\Cryp-Tor\wandb\run-20240329_220405-s82ool3r\logs\debug-internal.log
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_init.py:init():564] calling init triggers
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'epochs': 2}
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_init.py:init():614] starting backend
2024-03-29 22:04:05,293 INFO    MainThread:592 [wandb_init.py:init():618] setting up manager
2024-03-29 22:04:05,295 INFO    MainThread:592 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-29 22:04:05,297 INFO    MainThread:592 [wandb_init.py:init():624] backend started and connected
2024-03-29 22:04:05,298 INFO    MainThread:592 [wandb_init.py:init():716] updated telemetry
2024-03-29 22:04:05,333 INFO    MainThread:592 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-03-29 22:04:05,913 INFO    MainThread:592 [wandb_run.py:_on_init():2254] communicating current version
2024-03-29 22:04:06,243 INFO    MainThread:592 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-29 22:04:06,243 INFO    MainThread:592 [wandb_init.py:init():800] starting run threads in backend
2024-03-29 22:04:09,564 INFO    MainThread:592 [wandb_run.py:_console_start():2233] atexit reg
2024-03-29 22:04:09,564 INFO    MainThread:592 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-03-29 22:04:09,564 INFO    MainThread:592 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-03-29 22:04:09,564 INFO    MainThread:592 [wandb_run.py:_redirect():2178] Redirects installed.
2024-03-29 22:04:09,564 INFO    MainThread:592 [wandb_init.py:init():841] run started, returning control to user process
2024-03-29 22:05:45,993 ERROR   MainThread:592 [wandb_init.py:init():1188] Invalid project name 'unitnais/Cryp-Tor': cannot contain characters '/,\\,#,?,%,:', found '/'
Traceback (most recent call last):
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_init.py", line 1166, in init
    wi.setup(kwargs)
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_init.py", line 320, in setup
    settings._apply_init(kwargs)
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_settings.py", line 1859, in _apply_init
    self.update(init_settings, source=Source.INIT)
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_settings.py", line 1492, in update
    self.__dict__[key].update(settings.pop(key), source=source)
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_settings.py", line 573, in update
    self._value = self._validate(self._preprocess(value))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_settings.py", line 543, in _validate
    if not v(value):
           ^^^^^^^^
  File "C:\Users\Mateo-drr\AppData\Local\miniconda3\Lib\site-packages\wandb\sdk\wandb_settings.py", line 969, in _validate_project
    raise UsageError(
wandb.errors.UsageError: Invalid project name 'unitnais/Cryp-Tor': cannot contain characters '/,\\,#,?,%,:', found '/'
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Configure stats pid to 592
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Mateo-drr\.config\wandb\settings
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Loading settings from S:\Cryp-Tor\wandb\settings
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'cryptolsmt.py', 'program_abspath': 's:\\cryp-tor\\cryptolsmt.py', 'program': 's:\\cryp-tor\\cryptolsmt.py'}
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_init.py:_log_setup():524] Logging user logs to S:\Cryp-Tor\wandb\run-20240329_220707-egxynjl5\logs\debug.log
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_init.py:_log_setup():525] Logging internal logs to S:\Cryp-Tor\wandb\run-20240329_220707-egxynjl5\logs\debug-internal.log
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_init.py:init():564] calling init triggers
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'epochs': 2}
2024-03-29 22:07:07,907 INFO    MainThread:592 [wandb_init.py:init():609] wandb.init() called when a run is still active
2024-03-29 22:08:01,625 WARNING MsgRouterThr:592 [router.py:message_loop():77] message_loop has been closed
